ansible/roles/tilemaker/templates/bicycle/style.json

```diff
@@ -2348,6 +2348,24 @@
         "line-width": {"base": 1.3, "stops": [[15, 0.5], [20, 6]]}
       }
     },
+    {
+      "id": "poi-bike-track",
```

Comment:
A small nitpick: POI stands for "point of interest" and this isn't a point.

I will merge it and change the name.